lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_priority.rb

```diff
@@ -0,0 +1,43 @@
+module GovukPublishingComponents
+  module Presenters
+    class ContentBreadcrumbsBasedOnPriority
+      # keys are labels, values are the content_ids for the matching taxons
+      # Where multiple matching taxons are present, the top most one is the highest priority
+      #   and the bottom one the lowest priority
+      PRIORITY_TAXONS = {
+        education_coronavirus: "272308f4-05c8-4d0d-abc7-b7c2e3ccd249",
+        business_coronavirus:  "65666cdf-b177-4d79-9687-b9c32805e450",
+        coronavirus:           "5b7b9532-a775-4bd2-a3aa-6ce380184b6c"
```

Comment:
I _think_ we can leave the link to the top level coronavirus taxon out now, given there's a banner to here on every page - covered in the slack comment from Sam:
> my only reservation with Simon's approach is that a superbreadcrumb linking to the homepage is duplicating the site-wide banner